specs/overview.md

```diff
@@ -0,0 +1,158 @@
+# Optimism Overview
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Architecture Design Goals](#architecture-design-goals)
+- [Components](#components)
+  - [L1 Components](#l1-components)
+  - [L2 Components](#l2-components)
+  - [Transaction/Block Propagation](#transactionblock-propagation)
+- [Key Interactions In Depth](#key-interactions-in-depth)
+  - [Deposits](#deposits)
+  - [Block Derivation](#block-derivation)
+    - [Overview](#overview)
+    - [Epochs and the Sequencing Window](#epochs-and-the-sequencing-window)
+    - [Block Derivation Loop](#block-derivation-loop)
+  - [Engine API](#engine-api)
+  - [Sequencing](#sequencing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+This document is a high-level technical overview of the Optimism protocol. It aims to explain how the protocol works in an informal manner, and direct readers to other parts of the specification so that they may learn more.
+
+This document assumes you've read the [introduction](./introduction.md).
+
+## Architecture Design Goals
+
+- **Execution-Level EVM Equivalence:** The developer experience should be identical to L2 except where L2 introduces a fundamental difference.
+	- No special compiler.
+	- No unexpected gas costs.
+	- Transaction traces work out-of-the-box.
+	- All existing Ethereum tooling works - all you have to do is change the chain ID.
+- **Maximal compatibility with ETH1 nodes:** The implementation should minimize any differences with a vanilla Geth node, and leverage as many existing L1 standards as possible.
+	- The execution engine/rollup node use the ETH2 Engine API to build the canonical L2 chain.
+	- The execution engine leverages Geth's existing mempool and sync implementations, including snap sync.
+- **Minimize state and complexity:**
+	- Whenever possible, services contributing to the rollup infrastructure are stateless.
+	- Stateful services can recover to full operation from a fresh DB using the peer-to-peer network and on-chain sync mechanisms.
+	- Running a replica is as simple as running a Geth node.
+
+
+## Components
+
+![Components](./assets/components.svg)
+
+### L1 Components
+
+- **DepositFeed**: A feed of L2 transactions which originated as smart contract calls in the L1 state.
+	- The `DepositFeed` contract emits `TransactionDeposited` events, which the rollup driver reads in order to process deposits.
+	- Deposits are guaranteed to be reflected in the L2 state within the _sequencing window_.
```

Comment:
Actually the first L2 block that is based off the respective L1 block. Deposits also cannot be reordered.

Reply:
I think the above sentence is correct and fine to leave as-is, since at this stage the idea that L2 blocks belong to epoch derived from L1 blocks hasn't been introduced yet.